src/components/profile/Profile.vue

```diff
@@ -517,15 +524,13 @@ export default {
   color: var(--gray-50);
   border: 2px solid var(--gray-30);
 }
-.profile__section--editing {
-  border-color: var(--blue-60);
+.profile__section.profile__section--editing {
+  border: 2px solid var(--blue-60);
```

Comment:
Border is set in `.profile__section` (line 499 in your PR). Since we use `box-sizing: boder-box` this makes it jumpy. That's why there was: `border-color: transparent`.

Reply:
I understand this, but when everything is closed, we want there to be no border and we don't want to have to use negative margins. I've adjusted so that this shouldn't be a problem anymore.